src/components/referrals/ReferralsComponentLabel.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+    <li class="documentBox">
+        <font-awesome-icon :icon="classOfIcon" />
+        <h5 class="documentBox__title">{{ name }} {{ date }}</h5>
```

Comment:
Divide this into 2 tags. Name should be in h5 and date in different tag (not wrapped in h5)